test/basedriver/capabilities-specs.js

```diff
@@ -248,11 +248,18 @@ describe('caps', function () {
       }).should.deep.equal({hello: 'world', foo: 'bar'});
     });
 
-    it('should throw an exception if a standard capability (https://www.w3.org/TR/webdriver/#dfn-table-of-standard-capabilities) is prefixed', function () {
-      (() => processCapabilities({
+    it('should still accept prefixed caps even if they are standard capabilities (https://www.w3.org/TR/webdriver/#dfn-table-of-standard-capabilities)', function () {
+      processCapabilities({
         alwaysMatch: {'appium:platformName': 'Whatevz'},
         firstMatch: [{'appium:browserName': 'Anything'}],
-      })).should.throw(/standard capabilities/);
+      }).should.deep.equal({platformName: 'Whatevz', browserName: 'Anything'});
+    });
+
+    it('should prefer standard caps that are non-prefixed to prefixed', function () {
```

Comment:
it would also be good to have a test for 

```
"firstMatch": [{
  "appium:platformName": "iOS"
}],
"alwaysMatch": {
  "platformName": "Android"
}
```

case

Reply:
And with other prefixes as your original motivation?
```
"sauce:options": {
  "appiumVersion": "1.16.0"
}
```

Reply:
@mykola-mokhnach I just confirmed that having duplicate keys in firstMatch and alwaysMatch is forbidden.

From the documentation:
```
For each enumerable own property in secondary, run the following substeps:

Let name be the name of the property.

Let value be the result of getting a property named name from secondary.

Let primary value be the result of getting the property name from primary.

If primary value is not undefined, **return an error with error code invalid argument**.

Set a property on result with name name and value value.
```

(primary value refers to the value that came from `alwaysMatch` object)

Reply:
@KazuCocoa other non appium: prefixes just get ignored. The prefixes aren't stripped out.